dataStructures/test/queue_test.js

```diff
@@ -0,0 +1,95 @@
+import {expect} from 'chai'
+import Queue from '../structures/queue'
+
+describe('Queue', () => {
+  let queue
+  beforeEach(() => queue = new Queue())
+
+  it('is a function', () => {
+    expect(Queue).to.be.a('function')
+    expect(queue.enqueue).to.be.a('function')
+    expect(queue.dequeue).to.be.a('function')
+    expect(queue.front).to.be.a('function')
+    expect(queue.back).to.be.a('function')
+    expect(queue.isEmpty).to.be.a('function')
+    expect(queue.Length).to.be.a('function')
+  })
+
+  context('#enqueue', () => {
+    it('adds elements to a queue', () => {
+      queue.enqueue('hi')
+      expect(queue.content).to.eql(['hi'])
```

Comment:
It would be better to use the methods of queue to examine data, rather than exposing the data itself: `expect(queue.front()).to.equal('hi')`